common/src/main/java/org/apache/uniffle/common/metrics/GRPCMetrics.java

```diff
@@ -25,6 +25,13 @@
 import io.prometheus.client.Gauge;
 
 public abstract class GRPCMetrics {
+  // Grpc server internal executor metrics
+  public static final String GRPC_SERVER_EXECUTOR_ACTIVE_THREADS_TAG = "grpcServerExecutorActiveThreads";
```

Comment:
Why does our name contain `TAG`?

Reply:
`TAG` is an identifier of metric, like the `REGISTER_SHUFFLE_METHOD` in `ShuffleServerGrpcMetrics`.

In order to distinguish the name of `gaugeMap` and the name of the metricsManager's `gauge`, the TAG reservation is necessary

Reply:
`TAG` is too general, I feel it's a little inappropriate, could we give it a better name?

Reply:
GRPC_SERVER_EXECUTOR_ACTIVE_THREADS_TAG -> GRPC_SERVER_EXECUTOR_ACTIVE_THREADS_KEY ? 

This name wont be as the metric name exposed to rest api result, just be as the gaugeMap key

Reply:
OK.

Reply:
done